### ISSUE TITLE
SQL error on initial startup

### ISSUE THREAD

**Issue by half-adder** [org-roam/org-roam-ui#43]
I just installed this package (using doom emacs) and upon starting `org-roam-ui-mode`, I get the following error:
```
Error (websocket): in callback `on-open': EmacSQL had an unhandled condition: "near line 11: no such table: nodes"
```

**Comment by ThomasFKJorna** [comment]
Do you have org-roam-v2 installed, and have created at least one note?

**Comment by Lownin** [comment]
I am having this same issue.

> Error (websocket): in callback `on-open': EmacSQL had an unhandled condition: "near line 29: no such table: nodes"

I've only just tried to use org-roam-ui for the first time, so org-roam-ui hasn't worked for me in the past yet.

Some possibly relevant details:
* DOOM Emacs on macOS
* Clean break from v1 org-roam.  I switched to org-roam-v2 and didn't keep any of my old notes.
* Relatively vanilla install of DOOM Emacs
* I've set my org-roam directory in config.el to my iCloud drive with `(setq org-directory "~/Library/Mobile Documents/com~apple~CloudDocs/org")
`
* I _do_ have multiple notes.

**Comment by ThomasFKJorna** [comment]
Hmm strange. Are you sure that `org-roam` is also running? I.e. have your run `org-roam-setup`?

**Comment by Lownin** [comment]
`org-roam-setup` does not appear in my meta menu, but `org-roam`, `org-roam-insert` etc do.
Also, I can invoke the backlink buffer and that appears to be working.

**Comment by ThomasFKJorna** [comment]
I'm pretty sure `org-roam-setup` should be visible 

![image](https://user-images.githubusercontent.com/21983833/128428533-7da0529a-451a-447c-b4d3-b03c2fb033e4.png)

What I think the problem is is that this function is not called, and therefore `org-roam-db-location` or whatever is not properly set, which makes `org-roam-db-query` fail.

Do you use org-roam via the Doom +roam2 flag? I just had a look and they do some custom things

https://github.com/hlissner/doom-emacs/blob/f74debdea8042c3a19cf83781d9560bb467d86ce/modules/lang/org/contrib/roam2.el#L32-L67

This might prevent org-roam-ui from finding and using the database. Although it says it tries to init before querying it, but that's probably too late for us.

Could you do `SPC-;` `(org-roam-db-sync)` for me and then restart org-roam-ui?

**Comment by Lownin** [comment]
I did a dumb.

tl;dr: I was on roam1

Longer version:  I simplified my config to try to narrow down problems migrating to roam2.  I forgot I had some unpinning going on in my original config. I thought +roam in doom was giving me v2, but it was the unpinning. When I undid it and started over, I started over again in v1 without realizing.

I'm on v2 now and org-roam-ui is working well.

Thanks!

**Comment by ThomasFKJorna** [comment]
Glad to hear!
